src/Util/StaticRectorStrings.php

```diff
@@ -93,9 +93,9 @@ public static function removePrefixes(string $value, array $prefixesToRemove): s
      */
     public static function removeSuffixes(string $value, array $suffixesToRemove): string
     {
-        foreach ($suffixesToRemove as $prefixToRemove) {
-            if (Strings::endsWith($value, $prefixToRemove)) {
-                $value = Strings::substring($value, 0, -Strings::length($prefixToRemove));
+        foreach ($suffixesToRemove as $singleSuffixesToRemove) {
```

Comment:
Here it should be kept too, as suffixes to suffix

Reply:
is Inflector service need extract camel case with end "s" or "es" on first part to check it?

Reply:
Yes

Reply:
It seems we need to extends the `Doctrine\Inflector\Inflector` service as I realized it came from vendor.

Reply:
Great :+1: go for it

Reply:
implemented.